packages/grid/_modules_/grid/models/colDef/gridCheckboxSelection.tsx

```diff
@@ -17,7 +17,7 @@ export const gridCheckboxSelectionColDef: GridColDef = {
   disableClickEventBubbling: true,
   disableColumnMenu: true,
   valueGetter: (params) => params.api.getState().selection[params.id] !== undefined,
-  renderHeader: (params) => <GridHeaderCheckbox {...params} />,
+  renderHeader: () => <GridHeaderCheckbox />,
```

Comment:
why do you need to remove the params here?

Reply:
`GridHeaderCheckbox` component was not using any of the props (GridColumnHeaderParams) passed. So I think we need not type it,
```diff
- export const GridHeaderCheckbox: React.FC<GridColumnHeaderParams> = () =>
+ export const GridHeaderCheckbox = () => {
```
And also thus need to remove the `params` from here as well.